src/main.js

```diff
@@ -468,6 +474,8 @@ const CodingChatboxMainWindow = new Lang.Class({
             return container;
 
         let messageBecameVisibleHandler = Lang.bind(this, function() {
+            messageQueue.showNext();
```

Comment:
First to show the message bubble containing the text or content, and then to show the next pending bubble (whether that be an input box, pending message indicator or something else). We need to do it twice since the bubbles with pending messages are really two bubbles and not one.